airflow/models/dag.py

```diff
@@ -1892,6 +1892,7 @@ def create_dagrun(
         state: State,
         execution_date: Optional[datetime] = None,
         run_id: Optional[str] = None,
+        queued_at: Optional[datetime] = None,
```

Comment:
We shouldn't pass this here -- it's overriding the default to `DagRun()` with an explicit None/NULL value.